ocs_ci/deployment/helpers/rosa_prod_cluster_helpers.py

```diff
@@ -0,0 +1,133 @@
+"""
```

Comment:
maybe this file code can be move-in ocs_ci/utility/rosa.py or ocs_ci/utility/managedservice.py

Reply:
this functions are only for helping in deployment of production env clusters. That the reason this file resides in deployment/helpers

Reply:
I agree with Suchita that this should not be in helpers but as it contains a full class with defined purpose (to manage rosa on production) it should be placed in a modul dedicated to it like `ocs_ci/utility/rosa.py` or `ocs_ci/utility/rosa_production.py`.

Reply:
all the functions ( creating admin account, login to cluster, generating kubeconfig, generatin kubeadmin passwords ) are all one time activity which is needed to setup fully activated production cluster. 

@fbalak @suchita-g any other scenarios where we would like to use above functions?

Reply:
> This file contains `A base class for Managed Service ROSA Cluster in Production Environment`. According to this docstring on line 19 this isn't a helper function but a base class. In our hierarchy we have directories: `ocs` to contain modules and functions that use oc commands and `utility` which contain functions like this that handle other interfaces.

Changed doc string, now its `helper class`. deployment/helpers is specially for deployment purpose.

 What is an additional value of using `helpers` instead of `utility`? E.g. we have [helpers.py file](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/helpers.py) that contains 3756 lines but there is no clear description what it contains or what can be found there. I think that we shouldn't use `helpers` (at least for base classes) and that it brings unnecessary complexity.

Simplicity. [helpers.py file](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/helpers.py) already contains 3756 lines. Its ocean of functions.